zetaclient/bitcoin/txscript.go

```diff
@@ -0,0 +1,227 @@
+package bitcoin
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"strconv"
+
+	"github.com/btcsuite/btcd/btcjson"
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcutil"
+	"github.com/cosmos/btcutil/base58"
+	"github.com/zeta-chain/zetacore/common"
+	"github.com/zeta-chain/zetacore/common/bitcoin"
+	"golang.org/x/crypto/ripemd160"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	// Length of P2TR, P2WSH, P2WPKH, P2SH, P2PKH scripts
+	LengthScriptP2TR   = 34
+	LengthScriptP2WSH  = 34
+	LengthScriptP2WPKH = 22
+	LengthScriptP2SH   = 23
+	LengthScriptP2PKH  = 25
```

Comment:
Need documentation comment if the variable is public.

Would be good also to provide the reference how the values are determined

Reply:
https://github.com/zeta-chain/node/pull/1942/commits/2bcdfbc727c12612159ffbf008bb00505a996b4b